### PR TITLE
Add instructions for installing from marketplace

### LINKED ISSUE #50: Update documentation of the VSCode extension
> The extension is currently not published, so you will have to install it manually by downloading a `vsix` file.
https://effekt-lang.org/docs/getting-started#setting-up-vscode

This is wrong, we already have a published extension: https://marketplace.visualstudio.com/items?itemName=effekt-lang.effekt

### PR DESCRIPTION
**Pull request by marvinborner**
Closes #50 by adding helpful instructions.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -112,9 +112,22 @@ Hello
 
 ## Setting Up VSCode
 Effekt comes with a basic language server implementation (LSP).
-The [effekt-vscode repository](https://github.com/effekt-lang/effekt-vscode)
-The extension is currently not published, so you will have to install it
-manually by downloading a `vsix` file.
+Source code: [effekt-vscode repository](https://github.com/effekt-lang/effekt-vscode).
```
Comment (jiribenes) [inline docs/getting-started.md:115]: Nitpick:
```suggestion
The source code for the VSCode extension is available in the [`effekt-vscode` repository](https://github.com/effekt-lang/effekt-vscode).
```

(To be honest, I'm not sure it's too relevant for the users)

File: docs/getting-started.md [hunk 2]
```diff
@@ -112,9 +112,22 @@ Hello
...
+
+### Visual Studio Marketplace 
+
+You can install the extension from the Visual Studio Marketplace.
+
+1. Select `Preferences / Extensions` in the menu
+2. Type "effekt" in the search bar
+3. Click `Install`
```
Comment (jiribenes) [inline docs/getting-started.md:123]: Could we specify the name of the official extension in more detail to deter any name squatting/phishing attempts?The identifier is `effekt-lang.effekt` -- could we perhaps mention it?

Also, side note, is there something like automatic detection of file extensions? (When you open an `.effekt` file, does VSCode automatically recommend you the extension already?)

**Review by jiribenes — APPROVED** [review]
LGTM, have some minor nitpicks, but feel free to ship this already :shipit:

File: docs/getting-started.md (hunk 2 above)
Comment (marvinborner) [reply docs/getting-started.md:123]: Added the identifier. Don't know about the automatic detection, we should maybe create a separate issue for that.

**Pull request merged**